File: 0217-contains-duplicate/0217-contains-duplicate.java
import java.util.Arrays;

class Solution {
    public boolean containsDuplicate(int[] nums) {
        Arrays.sort(nums);
        System.out.println(nums);
        int duplicate = nums[0];
        for ( int i = 1 ; i < nums.length ; i++ ) {
            if ( nums[i] == duplicate ) {
                return true;
            } else {
                duplicate = nums[i];
            }
        }
        return false;
    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    const sortedArray = nums.sort((a,b)=>a-b)
    let unit = sortedArray[0];
    for ( let i = 1 ; i <= sortedArray.length ; i++ ) {
        if ( unit === sortedArray[i] ) {
            return true;
        }
        unit = sortedArray[i];
    }
    return false;
};